specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-11-01/netapp.json

```diff
@@ -0,0 +1,4890 @@
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - account updated",
+            "schema": {
+              "$ref": "#/definitions/netAppAccount"
+            }
+          },
+          "201": {
+            "description": "Account created",
+            "schema": {
+              "$ref": "#/definitions/netAppAccount"
+            }
+          },
+          "202": {
+            "description": "Accepted -- Create, update or delete request accepted; operation will complete asynchronously"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Accounts_CreateOrUpdate": {
+            "$ref": "examples/Accounts_CreateOrUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "NetApp Accounts"
+        ],
+        "operationId": "Accounts_Delete",
+        "summary": "Delete a NetApp account",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "description": "Delete the specified NetApp account",
+        "responses": {
+          "202": {
+            "description": "Accepted -- Create, update or delete request accepted; operation will complete asynchronously"
+          },
+          "204": {
+            "description": "NoContent -- Resource does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Accounts_Delete": {
+            "$ref": "examples/Accounts_Delete.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "NetApp Accounts"
+        ],
+        "operationId": "Accounts_Update",
+        "summary": "Update a NetApp account",
+        "description": "Patch the specified NetApp account",
+        "parameters": [
+          {
+            "name": "body",
+            "description": "NetApp Account object supplied in the body of the operation.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/netAppAccountPatch"
```

Comment:
Any reason to have two similar definitions? You also have: netAppAccount and if not mistaken the only difference is systemData which is readOnly so you should be ok to reuse it.

Reply:
Same as earlier comment this has been in from the beginning. Happy to change if it improves the spec. Only concern is if that would be a breaking change for SDK's generated?

Reply:
I cannot speak for the SDK generation process but I think it would as the object is changing, if it was like this in your previous version I guess you can continue with it but consider updating it in a future version.